include/wx/generic/infobar.h

```diff
@@ -96,10 +110,14 @@ class WXDLLIMPEXP_CORE wxInfoBarGeneric : public wxInfoBarBase
     // should be called when we're shown or hidden)
     void UpdateParent();
 
+    long m_style = 0;
```

Comment:
I don't think we need this one, we should store the style in `wxWindow::m_windowStyle`.